tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -393,9 +400,9 @@ class WskBasicTests extends TestHelpers with WskTestHelpers {
       action.create(name, Some(TestUtils.getTestActionFilename("wc.js")))
     }
 
-    wsk.action
-      .invoke(name, Map("payload" -> "one two three".toJson), result = true)
-      .stdout should include regex (""""count": 3""")
+    val result = wsk.action
+      .invoke(name, Map("payload" -> "one two three".toJson), blocking = true, result = true)
+    result.stdout should include regex (""""count":3""")
```

Comment:
this check is weaker now - it should be:
result.stdout.parseJson.asJsObject shouldBe JsObject("count" -> JsNumber(3))

Reply:
Done.